src/policies/conditions.ts

```diff
@@ -101,10 +101,10 @@ export class Condition {
   ];
 
   public static readonly SUPPORTED_CHAINS = [
-    'Ethereum Mainnet',
-    'Görli',
-    'Polygon Mainnet',
-    'Mumbai',
+    1,     // ethereum/mainnet
+    5,     // ethereum/goerli
+    137,   // polygon/mainnet
+    80001  // polygon/mumbai
```

Comment:
We could also use `ethers` to provide these values: https://docs.ethers.io/v5/api/providers/types/#providers-Network

Reply:
Nice idea but I'm having a difficult time making it work/

```js
private static readonly SUPPORTED_CHAIN_NAMES = [
    'mainnet',
    'goerli',
    'matic',
    'maticm'
];

 public static readonly SUPPORTED_CHAINS = Condition.SUPPORTED_CHAIN_NAMES.map(chainName =>
    ethers.providers.getNetwork(chainName).chainId
);
```
    
This seems to work only for `mainnet`, despite the docs stating that it works for other networks too.  Is this API meant only for connected providers?

Reply:
i have used this package before https://www.npmjs.com/package/evm-chains - it's excellent.
4K weekly downloads, it's the standard way of doing things

Reply:
After some more thought, I don't see any advantage to using a lookup by name string versus a chain ID.  Either way we're maintaining a hardcoded list of identifiers Chain IDs are less ambiguous than the network naming conventions implemented by the package maintainer.

Reply:
completely agree, everything internal should be chain id
i guess that package i posted is really for ui stuff, possibly useful on the demo so that a user knows they're not connected on mainnet